lib/uo_cli.rb

```diff
@@ -375,11 +376,13 @@ def opt_des_params
             "Example: uo des_params --sys-param path/to/sys_params.json --feature path/to/example_project.json\n", type: String, required: true, short: :f
 
           opt :model_type, "\nSelection for which kind of DES simulation to perform\n" \
-            "Valid choices: 'time_series'", type: String, default: 'time_series'
```

Comment:
This `default` entry was the source of the problem. The GMT cli supplies its own default values, and our overrides were confusing it. Removing this allows the GMT expected behavior to happen.